### DISCUSSION TITLE
Question about maxLFQ output and maxLFQ using the diann package

### DISCUSSION THREAD

**Discussion by ianchlee** [vdemichev/DiaNN#593] [Q&A]
I have received a DIA-NN output tsv from a collaborator and would like to run different analysis on the data. Is there any difference between the Gene.MaxLFQ and Gene.MaxLFQ.Unique column? Would I obtain the same maxLFQ result using the diann package using diann_maxlfq?

**Comment by vdemichev** [comment]
Gene.MaxLFQ.Unique is calculated using only precursors specific to the gene.
No, diann_maxlfq will not yield an identical result.

**Reply by ianchlee** [reply]
Thanks for the response. What about Gene.MaxLFQ? Would it be the same from diann_maxlfq? What would be their difference, and should we use Gene.MaxLFQ or diann_maxlfq to obtain maxLFQ normalised result?

**Reply by vdemichev** [reply]
diann_maxlfq uses a slighly different algorithm

**Reply by ianchlee** [reply]
I see. Should we therefore use the Gene.MaxLFQ result directly, or use diann_maxlfq on the Precursor.Normalised data?
